feat: add panels recommendations doc (#70)
perses/community-dashboards: docs/README.md

# Dashboard Design Recommendations

> We recommend following some standards when building dashboards, to keep the visualization looking good and making them better for usability. 

Reviewer: ```suggestion
We recommend following some standards when building dashboards, to keep the visualization looking good and making them better for usability. These are optional recommendations, there is no linting for this. 
If porting upstream existing dashboards, feel free to defer to original designs.
```

## Time Series Panels

> * For non duration metric panels (counter/gauge): Fully opaque charts:

Reviewer: ```suggestion
* For generic non-histogram metric panels (counter/gauge): Fully opaque area charts that give you visibility of the entire data even in darker backgrounds, and provide greater contrast when selecting result series as well.
```

Author: Hi thanks, added the recommendations just with minor changes, when you have the chance please review 🙏

```go
timeSeriesPanel.WithVisual(timeSeriesPanel.Visual{
				Display:      timeSeriesPanel.LineDisplay,
				ConnectNulls: false,
				LineWidth:    0.25,
				AreaOpacity:  1,
				Palette:      timeSeriesPanel.Palette{Mode: timeSeriesPanel.AutoMode},
			}),
```

> * For request Rate, error ratio panels: Stacked fully opaque charts:

Reviewer: ```suggestion
* For request rate, error rates, error ratio/percentage panels: Stacked fully opaque charts, as the opacity provides visual contrast, and stack allows the user to see result series data side by side, in relation to each other. This is particularly useful in cases where you want to visually be able to discern success cases (2xx) to error cases (4xx, 5xx).
```

```go
timeSeriesPanel.WithVisual(timeSeriesPanel.Visual{
				Display:      timeSeriesPanel.LineDisplay,
				ConnectNulls: false,
				LineWidth:    0.25,
				AreaOpacity:  1,
				Stack:        timeSeriesPanel.AllStack,
				Palette:      timeSeriesPanel.Palette{Mode: timeSeriesPanel.AutoMode},
			}),
```

> * For duration/quantile metric panels: Semi-opaque non stacked:

Reviewer: ```suggestion
* For duration/quantile metric panels or metrics panels where there is no benefit in seeing result series relations: Semi-opaque non-stacked, as that allows for seeing all of the resultant series at once, and gives the user the option to selectively view, but also allows the user to detect outliers within the data.
```
